src/engine/source/cmds/src/start.cpp

```diff
@@ -138,6 +142,22 @@ void runStart(ConfHandler confManager)
               logConfig.flushInterval);
     LOG_INFO("Logging initialized.");
 
+    //TZDB Install
+    {
+        auto install = confManager->get<std::string>("server.tzdb_path");
+        date::set_install(install);
+        try
+        {
+            auto& tzdb  = date::reload_tzdb();
+        }
+        catch (const std::exception& e)
+        {
+            LOG_ERROR("An error occurred while reload the tzdb: {}.", utils::getExceptionStack(e));
+            exitHandler.execute();
+            return;
+        }
+    }
+
```

Comment:
This block is part of the engine initialization, it should be inside the try, just before or inside the HLP initialization, and it should throw a runtime_exception describing the error without the return.